test/stdlib/Enumerator_test.rb

```diff
@@ -25,6 +25,13 @@ class EnumeratorSingletonTest < Test::Unit::TestCase
 
   testing "singleton(::Enumerator)"
 
+  def test_produce
+    assert_send_type(
+      "(Integer initial) { (Integer) -> Integer } -> Enumerator[Integer, untyped]",
+      Enumerator, :produce, 1, &:succ
+    )
+  end
```

Comment:
Sorry, my PR is broken now. By mistake, I ran `rake test` on my local (not `rake stdlib_test`)... I'll check what happens tomorrow.

Reply:
Fixed on my local. But CI has still been broken by SEGV. I guess it's not related to this change.